pimpmycause/pimpmycause/templates/index.html

```diff
@@ -362,7 +362,7 @@ <h4 class="pnl__title">Sign up to our newsletter</h4>
                                     <div class="medium-6 large-6 small-12 columns">
                                         <input type="email" name="MERGE0" placeholder="Email Address" required>
                                     </div>
-                                    <div class="medium-6 large-6 small-12 columns">
+                                    <div class="medium-6 large-6 small-12 columns sign-up">
```

Comment:
Same here, please make the changes in the sass file and we already have the rule targeting this button in here: https://github.com/womenhackfornonprofits/pimp-my-cause-django/blob/master/frontend/src/sass/pages/_home.scss#L118, please make the button center perfectly in the center even if we increase the button size. You can use flex for this: https://philipwalton.github.io/solved-by-flexbox/demos/vertical-centering/